modules/nw-sriov-configure-exclude-topology-manager.adoc

```diff
@@ -0,0 +1,215 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-configure-exclude-topology-manager_{context}"]
+= Excluding SR-IOV network topology for NUMA-aware scheduling 
+
+To exclude advertising the SR-IOV network resource's Non-Uniform Memory Access (NUMA) node to the Topology Manager, you can configure the `excludeTopology` specification in the `SriovNetworkNodePolicy` custom resource. Use this configuration for more flexible SR-IOV network deployments during NUMA-aware pod scheduling.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have configured the CPU Manager policy to `static`. For more information about CPU Manager, see the _Additional resources_ section.
+* You have configured the Topology Manager policy to `single-numa-node`.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. Create the `SriovNetworkNodePolicy` CR:
+
+.. Save the following YAML in the `sriov-policy-numa.yaml` file, replacing values in the YAML to match your environment:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: <policy_name>
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: sriovnuma0 <1>
+  nodeSelector:
+    kubernetes.io/hostname: <node_name>
+  numVfs: <number_of_Vfs>
+  nicSelector: <2>
+    vendor: "<vendor_ID>"
+    deviceID: "<device_ID>"
+  deviceType: netdevice
+  excludeTopology: true <3>
+----
+<1> The resource name of the SR-IOV network device plugin. This YAML uses a sample `resourceName` value.
+<2> Identify the device for the Operator to configure by using the NIC selector.
+<3> To exclude advertising the NUMA node for the SR-IOV network resource to the Topology Manager, set the value to `true`. The default value is `false`.
+
+[NOTE]
+====
+If multiple `SriovNetworkNodePolicy` resources target the same SR-IOV network resource, the `SriovNetworkNodePolicy` resources must have the same value for the `excludeTopology` specification. Otherwise, the conflicting policy is rejected. 
+====
+
+
+.. Create the `SriovNetworkNodePolicy` resource by running the following command:
```

Comment:
This is appearing as the second sub-step `a)` in the build preview, and also throws off the remaining steps beneath it. If the `[NOTE]` above needs to be nested under the callouts sub-step, add a `+` on line 45.